### DISCUSSION TITLE
page encryption (closed)

### DISCUSSION THREAD

**Discussion by ghost** [CaiJimmy/hugo-theme-stack#642] [Ideas]
hey @CaiJimmy, i just found and fall in love with this theme bcz it's dead simple to set up, i have no idea of programming so i don't know what i am talking about really, so there is this
https://github.com/Li4n0/hugo_encryptor
https://github.com/Izumiko/hugo-encrypt
Demo:
https://0n0.fun/post/2019/03/this-is-hugo-encryptor/

not necessary but it would really be cool to have it :D

**Comment by CaiJimmy** [comment]
Sorry, I won't add more features to this theme 😥.

**Reply by ghost** [reply]
oh, np <3
didn't read the last post
